packages/site/.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Done! It had to be a '.cjs' file otherwise it threw with an error, but the eslint file in components is also a cjs file so it's more consistent than a JSON file.

And agree, when we upgrade to 9 we can overhaul everything to be in line.